src/components/30-organisms/commercial-hero-banner/ui.test.js

```diff
@@ -0,0 +1,203 @@
+import { Selector } from 'testcafe';
+
+const host = process.env.TEST_HOST_STORYBOOK_URL || 'http://localhost:9999';
+
+fixture('Commercial Hero Banner - basic functionality').page(
+  `${host}/iframe.html?id=organisms-commercial-hero-banner--commercial-hero-banner`
+);
+
+const TAG = 'axa-commercial-hero-banner';
+const ROOT_CLASS = '.o-commercial-hero-banner';
+
+test('should render commercial-hero-banner', async t => {
+  const $axaElem = await Selector(TAG);
+  await t.expect($axaElem.exists).ok();
+  const $axaElemShadow = await Selector(
+    () => document.querySelector('axa-commercial-hero-banner').shadowRoot
+  );
+  const $axaElemShadowEl = await $axaElemShadow.find(ROOT_CLASS);
+  await t.expect($axaElemShadowEl.exists).ok();
+});
+
+fixture('Commercial Hero Banner - Dark Mode').page(
+  `${host}/iframe.html?id=organisms-commercial-hero-banner--commercial-hero-banner&knob-Variant=dark&knob-Src=https://d5cplpsrt2s33.cloudfront.net/m/24c1b33e4e8ceda1/WIDE_1440_560_X2-hero_kv_neu_kv_breit_web.jpg`
+);
+
+test('should render in dark mode', async t => {
+  const $axaElem = await Selector(TAG);
+  await t.expect($axaElem.getAttribute('variant')).eql('dark');
+  const $axaElemShadow = await Selector(
+    () => document.querySelector('axa-commercial-hero-banner').shadowRoot
+  );
+
+  const container = $axaElemShadow.find('.o-commercial-hero-banner__container');
+  await t.expect(container.visible).ok();
+  await t
+    .expect(container.hasClass('o-commercial-hero-banner__container--dark'))
+    .ok();
+
+  await _assertDarkBackgroundGradient(t, container);
+});
+
+async function _assertDarkBackgroundGradient(t, container) {
+  await t
+    .expect(container.getStyleProperty('background-image'))
+    .eql(
+      'linear-gradient(270deg, rgba(0, 0, 0, 0) 30%, rgba(0, 0, 0, 0.8) 60%, rgb(0, 0, 0) 100%)'
+    );
+}
+
+fixture('Commercial Hero Banner - Content')
+  .page(
+    `${host}/iframe.html?id=organisms-commercial-hero-banner--commercial-hero-banner&knob-Variant=light&knob-Src=https://d5cplpsrt2s33.cloudfront.net/m/24c1b33e4e8ceda1/WIDE_1440_560_X2-hero_kv_neu_kv_breit_web.jpg`
+  )
+  .beforeEach(async t => {
+    await t.maximizeWindow();
+  });
+
+test('should render in light mode with background image position bottom left', async t => {
```

Comment:
The test description says "image position bottom left", but it is not tested here?

Reply:
Good find, I removed that option and forgot to adapt this test.